packages/jest-stylelint-runner/__mocks__/css-functions-list.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+
+// As we're running tests in `jsdom` environment, the internal logic
+// of this package `css-functions-list` relies on some properties in
+// the `document` object that are not available.
+// Therefore, to keep things simple, we just mock the module, which
+// returns the location of a JSON file.
+const cssFunctionsListJsonFilePath = path.join(
+  path.dirname(require.resolve('css-functions-list')),
+  'index.json'
+);
+
+const location = cssFunctionsListJsonFilePath;
```

Comment:
Kind of "ugly" but it's the "simplest" solution I can think of.

The original error is: `The URL must be of scheme file`.

If we look at the `css-functions-list` package, we can see how it's implemented:

<img width="675" alt="image" src="https://user-images.githubusercontent.com/1110551/179192299-fdc595d2-61a7-4b5a-91ef-ee164020d40f.png">

So wait, that's it? Where is this error coming from?

Well, the screenshot shows the ESM version, if we look at the CJS version we see this:

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/1110551/179192538-3787f5f2-b571-4a08-a23b-b5c7f1b14067.png">

Notice how it's checking for the existence of the global object `document`.

Since the tests in appkit run in `jsdom` environment, this `document` object is defined but with very limited properties.
In fact, the `document.currentScript` and `document.baseURI` properties are not defined, which is what causes the `The URL must be of scheme file` error.

Therefore, let's just mock this module.

On another note, we might want to look into our testing setup in appkit to maybe have 2 group of tests: one for `jsdom` environments and another one for `node` environments, as we do have some packages that are Node-only packages.